data/domain/world-2/votes.tsx

```diff
@@ -0,0 +1,101 @@
+import { Domain, RawData } from "../base/domain";
+import { Equinox } from "../equinox";
+import { Item } from "../items";
+
+export class Votes extends Domain {
+    multiFromEquinox: number = 0;
+    bonuses: VoteBonus[] = [];
+    currentBonusIndex: number = 0;
+    ongoingVoteTopOptionIndex: number = 0;
+    ongoingVoteMiddleOptionIndex: number = 0;
+    ongoingVoteBottomOptionIndex: number = 0;
+
+    ongoingVoteTopOptionPercent: number = 0;
+    ongoingVoteMiddleOptionPercent: number = 0;
+    ongoingVoteBottomOptionPercent: number = 0;
+
+    getRawKeys(): RawData[] {
+        return [
+            { key: "servervars", perPlayer: false, default: [] },
```

Comment:
I totally forgot how `serverVars` look but looking at line 29 where you convert it to a record, shouldn't the default be `{}`?

```suggestion
            { key: "servervars", perPlayer: false, default: {} },
```